components/admin_console/plugin_management/plugin_management.tsx

```diff
@@ -646,35 +694,40 @@ export default class PluginManagement extends AdminSettings {
 
     handleRemove = async () => {
         this.setState({lastMessage: null, serverError: null});
-        const {error} = await this.props.actions.removePlugin(this.state.removing);
-        this.setState({removing: null});
+        if (this.state.removing) {
```

Comment:
Is this just a null check? This might be changing logic since it's also checking if it evaluates to true as well.